testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -547,7 +547,7 @@
   if verb == 'be' && value == ''
     raise "Output has more than one line: #{output}" unless output.split("\n").length == 1
   elsif verb == 'be'
-    raise "Output value not found : #{output}" unless output.split("\n") > 1
+    raise "Output value not found : #{output}" unless output.split("\n").length() > 1
```

Comment:
You don't need the parens, just `.length` would do.

Reply:
right... for next resync between branches